mbed_host_tests/__init__.py

```diff
@@ -278,8 +278,8 @@ def init_host_test_cli_params():
                       action="store_true",
                       help='Skips use of reset plugin. Note: target will not be reset')
 
-    parser.add_option('-P', '--pooling-timeout',
-                      dest='pooling_timeout',
+    parser.add_option('-P', '--polling-timeout',
+                      dest='polling_timeout',
                       default=60,
                       metavar="NUMBER",
                       type="int",
```

Comment:
Maybe update the help text here to reflect that its used for local and remote device readiness